tests/src/model/pages/containers-page.ts

```diff
@@ -84,4 +110,11 @@ export class ContainersPage extends MainPage {
     await this.page.getByRole('button', { name: `Create Pod` }).click();
     return new CreatePodsPage(this.page);
   }
+
+  async pruneContainers(): Promise<ContainersPage> {
+    await this.pruneContainersButton.click();
+    await this.pruneConfirmationButton.waitFor({ state: 'visible', timeout: 3000 });
+    await this.pruneConfirmationButton.click();
```

Comment:
Can you please use `handleConfirmationDialog` as is used here: https://github.com/containers/podman-desktop/pull/5572/files#diff-4b11f9fa58190b54db2662ae7038e089037f9df7d2fa8aa6aa8104572920d64eR69 ?

Reply:
done